common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -1381,6 +1381,8 @@ remove_class_things_effect = {
 		if = { limit = { has_character_flag = prefers_talon_form 		} clr_character_flag = prefers_talon_form }
 		if = { limit = { has_character_flag = prefers_stag_form 		} clr_character_flag = prefers_stag_form }
 	}
+	### Moon Priestess ###
+	if = { limit = { trait = subclass_moon_priest 		} remove_trait = subclass_moon_priest 	}
```

Comment:
Conflicted lines. I moved them to wc_class_scripted_effects.txt. I'll implement your changes there myself.

Reply:
Done